File: Primeras practicas/calculo.cpp
#include <iostream>
#include <math.h>

using namespace std;

int main(){
    double i=1,seno,coseno,tangente,logaritmo,raiz;
    while(i<1000000000000000000000000000000000000){
        seno+=sin(i);
        coseno+=cos(i);
        tangente+=tan(i);
        logaritmo+=log(i);
        raiz+=sqrt(i);
        i++;
    }
    return 0;
}

File: Primeras practicas/calculo.java
class calculo 
{ 
    
    public static void main(String args[]) 
    { 
        double i=0,seno=0,coseno=0,tangente=0,logaritmo=0,raiz=0;
        while(i<10000000000000000000){
            seno+=Math.sin(i);
            coseno+=Math.cos(i);
            tangente+=Math.tan(i);
            logaritmo+=Math.log(i);
            raiz+=Math.sqrt(i);
            i++;
        } 
    } 
} 